src/main/java/pl/edu/icm/pl/mxrdr/extension/workflow/MxrdrWorkflowStepSPI.java

```diff
@@ -1,5 +1,6 @@
 package pl.edu.icm.pl.mxrdr.extension.workflow;
```

Comment:
One general remark: we have in ICM some [coding conventions](https://wiki.yadda.icm.edu.pl/yadda/Public:Coding_conventions) which contain (among others) the requirement to put the elements of the class in certain order and under certain headings.